hack/unwanted-dependencies.json

```diff
@@ -180,21 +148,51 @@
       "go.opencensus.io": [
         "github.com/Microsoft/hcsshim"
       ],
+      "google.golang.org/appengine": [
+        "github.com/prometheus/client_golang"
+      ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
         "github.com/antlr4-go/antlr/v4",
+        "github.com/cilium/ebpf",
+        "github.com/containerd/cgroups/v3",
         "github.com/google/cel-go",
         "k8s.io/mount-utils"
       ],
-      "google.golang.org/appengine": [
-        "github.com/prometheus/client_golang"
+      "github.com/gorilla/mux": [
```

Comment:
looks like github.com/gorilla/mux unarchived itself in https://github.com/gorilla/mux/pull/713, can drop this from the unwanted deps list in spec and drop this entry